conftest.py

```diff
@@ -1,6 +1,13 @@
 import pytest
 
 
+def pytest_configure(config):
+    # register an additional marker
+    config.addinivalue_line(
+        "markers", "pandoc_version_sensitive: marks tests that require a specific version of pandoc to pass"
+    )
```

Comment:
@epogrebnyak is this a good description of pytest.mark.pandoc_version_sensitive?

Was getting warning without https://travis-ci.com/manubot/manubot/jobs/249635218#L865